wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -460,3 +487,44 @@ definitions:
         type: boolean
       filename:
         type: string
+  CDRStatistics:
+    properties:
+      items:
+        items:
+          $ref: '#/definitions/CDRStatistic'
+        type: array
+      total:
+        type: integer
+    type: object
```

Comment:
Move `type: object` to the beginning of the class
why: be like other definition